http/src/v0/swarm.rs

```diff
@@ -0,0 +1,103 @@
+use serde::{Deserialize, Serialize};
+use std::collections::BTreeMap;
+
+#[derive(Debug, Deserialize)]
+pub struct ConnectQuery {
+    arg: String,
+}
+
+#[derive(Debug, Serialize)]
+#[serde(rename_all = "PascalCase")]
+pub struct ConnectResponse {
+    strings: Vec<String>,
+}
+
+pub async fn connect(query: ConnectQuery) -> Result<impl warp::Reply, std::convert::Infallible> {
```

Comment:
Offtopic: I was expecting the lints to come only from `ubuntu-latest`, do you understand why we see them from all three platforms?

Reply:
it's a warning emitted from the compiler, not from clippy/rustfmt